libraries/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -24,6 +24,12 @@ import type { SelectionParameterSet } from '../parsing/SelectionParameterSet';
 import type { RushConfigurationProject } from '../../api/RushConfigurationProject';
 import type { Subspace } from '../../api/Subspace';
 
+interface ISubspaceSelectedProjectsMetadata {
```

Comment:
```ts
/** 
  * Temporary data structure used by `BaseInstallAction.runAsync()`
  */
```
`ISubspaceSelectedProjectsMetadata` is an odd name, since that sounds like metadata of the selected projects, but it's really data about the subspace, and it's not really "metadata" -- that is extra details being attached to regular data.  A better name might be`ISubspaceInstallation`  or `ISubspaceInstall` since this data structure describes a planned invocation of `pnpm install` for a given subspace.